ui/styles.css

```
@@ -0,0 +1,214 @@
+  gap: 16px;
+  padding: 4px 0;
+  border-bottom: 1px solid var(--ebay-border);
+}
+
+.field-label {
+  color: var(--ebay-text-soft);
+}
+
+.field-value {
+  text-align: right;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)